### DISCUSSION TITLE
web页面 无法在pc端浏览器访问

### DISCUSSION THREAD

**Discussion by lamboxr** [AlistGo/alist#6101] [Invalid question]
24年2月25日 linux 通过脚本安装最新版本，昨天刚刚安装的时候，可以通过局域网pc浏览器访问（chrome edge firefox）, 今天再访问就是一片空白，服务是好的，局域网可以用手机chrome访问，卸载了重新在线安装还是只能手机浏览器打开web页面，pc浏览器一片空白，

**Comment by lamboxr** [comment]
搞了2个小时搞不动，才发的帖子，一发完帖子，好了，，，，，囧

**Comment by fzlychee2019** [comment]
和楼主一样，这两天ALIST突然出现这种问题，只能局域网访问，远程访问都是空白。

**Reply by lamboxr** [reply]
https://github.com/alist-org/alist/issues/6100#issuecomment-1964071872
解决方案在此
